src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -39,27 +39,30 @@ class ProjectModulesController(
 }
 
 data class ProjectModuleEventSession(
-    val id: EventId,
-    val startTime: Instant?,
     val endTime: Instant?,
+    val id: EventId,
     val meetingUrl: URI?,
     val recordingUrl: URI?,
     val slidesUrl: URI?,
+    val startTime: Instant?,
+    val type: EventType,
 ) {
   constructor(
-      model: EventModel
+      model: EventModel,
+      type: EventType,
   ) : this(
-      id = model.id,
-      startTime = model.startTime,
       endTime = model.endTime,
+      id = model.id,
       meetingUrl = model.meetingUrl,
       recordingUrl = model.recordingUrl,
       slidesUrl = model.slidesUrl,
+      startTime = model.startTime,
+      type = type,
   )
```

Comment:
Besides the addition of `type`, the rest of the changes are alphabetizing the arguments.